tests/cuda_custom_call/Makefile

```diff
@@ -0,0 +1,19 @@
+# This Makefile is not used by Bazel for this test, it is intended to serve as
+# documentation of build instructions for JAX users that are not using Bazel to
+# build their custom call code. For that reason, this Makefile is likely subject
+# to bitrot over time. Please file a JAX issue on GitHub if typing "make" in
+# this directory no longer runs the test to completion.
+NVCC = nvcc
+NVCCFLAGS += -I$(shell python -c 'from jax.extend import ffi; print(ffi.include_dir())')
+NVCCFLAGS += -arch native
+# since the file extension is .cu.cc, tell NVCC explicitly to treat it as .cu
```

Comment:
Than, why do we need to end the filename as .cc?

Reply:
Such is the Google convention I believe, I couldn't use the normal Bazel rules to compile otherwise.